olegbetsun/booking_up_for_beauty.go

```diff
@@ -0,0 +1,52 @@
+package booking
+
+import (
+	"time"
+	"strconv"
+	"fmt"
+	)
+
+func ParseDate(layout string, date string) time.Time{
```

Comment:
This function doesn't need to be exposed IMHO.

Reply:
I guess what David means is method name should start with lower case (parseDate). This is a comment for both of you :-).